src/main/java/com/dreammaster/gthandler/recipes/LatheRecipes.java

```diff
@@ -14,215 +17,211 @@
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs2", 1L, 7),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 0),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 1),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 2),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 3),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 4),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 5),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 6),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
-            GT_Values.RA.addLatheRecipe(
-                    GT_ModHandler.getModItem(Forestry.ID, "slabs3", 1L, 7),
-                    new ItemStack(Items.bowl, 1),
-                    GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 1),
-                    50,
-                    8);
+
+            GT_Values.RA.stdBuilder().itemInputs(GT_ModHandler.getModItem(Forestry.ID, "slabs1", 1L, 0))
```

Comment:
this probably doesnt work. (I already have lots of errors about these slabs not existing in my gregtech logs without this PR)